bin/serverless

```diff
@@ -7,10 +7,9 @@ let argv = require('minimist')(process.argv.slice(2));
 if (argv.debug) process.env.DEBUG = '*';
 
 let S           = require('../lib/Serverless'),
-    SUtils      = require('../lib/utils/index'),
-    serverless  = new S({
-      interactive: typeof process.env.CI !== 'undefined' ? false : true,
-      projectPath: SUtils.getProjectPath(process.cwd())
+    Project     = require('../lib/ServerlessProject'),
```

Comment:
If used as a npm module, people have to require two separate classes from it?  Then add one in another?  This isn't the out-of-the-box experience I want to provide.

I wonder if there is a way we could merge the Serverless and Project classes together.